apis/otel/v1alpha1/defaults.go

```diff
@@ -397,7 +397,181 @@ service:
 
 	defaultGatewayCPU    = "4"
 	defaultGatewayMemory = "8Gi"
-
+	defaultGatewayConfig = `
+    exporters:
+      sapm:
+        access_token: ${SPLUNK_ACCESS_TOKEN}
+        endpoint: https://ingest.${SPLUNK_REALM}.signalfx.com/v2/trace
+      signalfx:
+        access_token: ${SPLUNK_ACCESS_TOKEN}
+        api_url: https://api.${SPLUNK_REALM}.signalfx.com
+        ingest_url: https://ingest.${SPLUNK_REALM}.signalfx.com
+    extensions:
+      health_check: null
```

Comment:
Pardon my ignorance, is it normal to have no health checks?

Reply:
This normal. "health_check: null" and "health_check: " are the same here. Removing nulls where possible for readability.

Reply:
ah it's ok no problem.